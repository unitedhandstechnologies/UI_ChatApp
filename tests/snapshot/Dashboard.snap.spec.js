import 'react-native';
import React from 'react';
import renderer from 'react-test-renderer';
import Dashboard from '../../src/containers/Dashboard';
import store from '../../src/store';
import {Provider} from 'react-redux';

describe('Dashboard screen', () => {
  it('Dashboard screen', () => {
    const tree = renderer
      .create(
        <Provider store={store}>
          <Dashboard />
        </Provider>,
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
