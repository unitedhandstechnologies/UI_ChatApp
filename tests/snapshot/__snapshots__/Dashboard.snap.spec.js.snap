// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Dashboard screen Dashboard screen 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <View
    accessible={true}
    collapsable={false}
    focusable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "opacity": 1,
      }
    }
  >
    <Text
      allowFontScaling={false}
      style={
        Array [
          Object {
            "color": undefined,
            "fontSize": 36,
          },
          Object {
            "color": "#4A4A4A",
          },
          Object {
            "fontFamily": "FontAwesome",
            "fontStyle": "normal",
            "fontWeight": "normal",
          },
          Object {},
        ]
      }
    >
      ÔÄç
    </Text>
  </View>
  <Text
    style={
      Object {
        "fontSize": 25,
      }
    }
  >
    Welcome To Dashboard!!!
  </Text>
  <Text
    style={
      Object {
        "fontSize": 25,
      }
    }
  >
    Login as: 
  </Text>
</View>
`;
